pkg/flag/kubernetes_flags.go

```diff
@@ -173,33 +173,29 @@ func (f *K8sFlagGroup) Flags() []Flagger {
 	}
 }
 
-func (f *K8sFlagGroup) ToOptions() (K8sOptions, error) {
-	if err := parseFlags(f); err != nil {
-		return K8sOptions{}, err
-	}
-
+func (f *K8sFlagGroup) ToOptions(opts *Options) error {
 	tolerations, err := optionToTolerations(f.Tolerations.Value())
 	if err != nil {
-		return K8sOptions{}, err
+		return err
 	}
 
 	exludeNodeLabels := make(map[string]string)
 	exludeNodes := f.ExcludeNodes.Value()
 	for _, exludeNodeValue := range exludeNodes {
 		excludeNodeParts := strings.Split(exludeNodeValue, ":")
 		if len(excludeNodeParts) != 2 {
-			return K8sOptions{}, fmt.Errorf("exclude node %s must be a key:value", exludeNodeValue)
+			return fmt.Errorf("exclude node %s must be a key:value", exludeNodeValue)
```

Comment:
nit:
Perhaps we want to change it here (and bellow in this fucntion) to `xerror` right away

Reply:
Updated https://github.com/aquasecurity/trivy/pull/8718/commits/3add90a1af594bdd1284e1a114b6b9d5b45741c1